services/cards-publication/src/test/java/org/opfab/cards/publication/services/CardProcessServiceShould.java

```diff
@@ -734,6 +757,32 @@ void GIVEN_a_card_with_representative_dummyUser_WHEN_wrongUser_send_the_card_THE
                 Assertions.assertThat(checkCardCount(0)).isTrue();
         }
 
+        @Test
+        void GIVEN_a_card_with_representative_dummyUser_WHEN_card_is_send_with_a_login_case_different_than_representative_THEN_card_is_accepted() {
+
+                User user = new User();
+                user.setLogin("DummyUser");
+                CurrentUserWithPerimeters sameUser = new CurrentUserWithPerimeters();
+                sameUser.setUserData(user);
+
+                ComputedPerimeter cp = new ComputedPerimeter();
+                cp.setProcess("PROCESS_CARD_USER");
+                cp.setState("state1");
+                cp.setRights(RightsEnum.RECEIVEANDWRITE);
+                List<ComputedPerimeter> list = new ArrayList<>();
+                list.add(cp);
+                sameUser.setComputedPerimeters(list);
+
+                CardPublicationData card = generateOneCard("IGNORED_PUBLISHER");
+                card.setPublisherType(PublisherTypeEnum.EXTERNAL);
+                card.setRepresentativeType(PublisherTypeEnum.EXTERNAL);
+                card.setRepresentative(currentUserWithPerimeters.getUserData().getLogin());
```

Comment:
Should be for example card.setRepresentative("DUMMYUSER"), in your test login has the same case as representative

Reply:
resolved